lib/guidance.rb

```diff
@@ -0,0 +1,25 @@
+module Guidance
+  DOCUMENT_TYPES = %w[
```

Comment:
Can we also make sure `collections` uses this list from navigation helpers? That way we can delete https://github.com/alphagov/collections/blob/29a0085996311fa7e650dfd65eb805007c2cead9/app/models/rummager_search.rb#L4 and keep just one list of document types until we migrate this to the content store and search